.travis.yml

```diff
@@ -14,16 +14,21 @@ install:
   - pip install flake8 markdown pytest-cov -r requirements.txt -r test_requirements.txt
 before_script:
   - if [ "$TRAVIS_PYTHON_VERSION" = "3.7" ]; then
-      pip install black;
-      black --check .;
+      pip install black ;
+      black --check . ;
     fi
   # stop the build if there are Python syntax errors or undefined names
   - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
   # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
   - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
   - psql -c 'create database webpy;' -U postgres
   - mysql --user=root -e "create user 'scott'@'localhost' identified by 'tiger'; create database webpy; grant all privileges on webpy.* to 'scott'@'localhost' with grant option;"
-script: pytest --cov --cov-report=term-missing --cov-report=xml
+script:
+  - pytest --cov --cov-report=term-missing --cov-report=xml
+  - if [ "$TRAVIS_PYTHON_VERSION" = "2.7" ]; then
+      pip install twisted ;
+      pytest --doctest-modules --ignore=experimental/pwt.py --ignore=web/webopenid.py . || true ;
```

Comment:
We should run doctests for `web/*.py`, not `tests/*` and `experimental/*`.
Could you help fix this? Sorry i didn't make it clear in original issue.

Reply:
Done.